manifests/profile/hathitrust/apache.pp

```diff
@@ -110,6 +110,7 @@
   class { 'apache::mod::php':
     extensions => ['.php','.phtml']
   }
+  class { 'apache::mod::proxy': }
```

Comment:
The tests throw a bunch of errors without this :shrug: I suspect the `proxy_pass` block below pulled it in, and however it's being used elsewhere (probably via RewriteRules with the P flag) doesn't.